zerver/lib/webhooks/git.py

```diff
@@ -37,15 +39,25 @@
 
 def get_push_commits_event_message(user_name, compare_url, branch_name, commits_data, is_truncated=False):
     # type: (Text, Optional[Text], Text, List[Dict[str, Any]], Optional[bool]) -> Text
+
+    committers = get_all_committers(commits_data)
+    committers_details_extension = "{}({})".format(committers.items()[0][0], committers.items()[0][1])
```

Comment:
`committers_details_extension = "{}({})".format(*committers.items()[0])` should work and its more readable IMHO.